### PR TITLE
Centralize frontend API endpoints and update with versioned apis

### PR DESCRIPTION
**Pull request by mamy-CS**
Update Frontend to use Tornjak v1 API

### CODE REVIEW

File: frontend/src/components/tornjak-api-helpers.tsx [hunk 1]
```diff
@@ -147,72 +109,32 @@ class TornjakApi extends Component<TornjakApiProp, TornjakApiState> {
...
-      .catch((error) => showResponseToast(error, { caption: "Could not populate tornjak agent info." }))
-  }
-
   // populateLocalTornjakAgentInfo returns tornjak info of requested agents including cluster name and selector
   populateLocalTornjakAgentInfo = (
     agentworkloadSelectorInfoFunc: (globalAgentsWorkLoadAttestorInfo: AgentsWorkLoadAttestorInfo[]) => void,
     inputData: string
   ) => {
-    axios.post(GetApiServerUri("/api/tornjak/agents/list"), inputData, { crossdomain: true })
+    axios.post(GetApiServerUri(apiEndpoints.tornjakAgentsApi), inputData, { crossdomain: true })
```
Comment (maia-iyer) [inline frontend/src/components/tornjak-api-helpers.tsx:125]: shouldn't this be a get?

**Review by maia-iyer — COMMENTED** [review]
Hello just one comment: I think a different method is necessary? Let me know if that's incorrect

File: frontend/src/components/tornjak-api-helpers.tsx (hunk 1 above)
Comment (mamy-CS) [reply frontend/src/components/tornjak-api-helpers.tsx:125]: A different name is probably necessary there but that function updates agents including their cluster and selector info. There are some updates I need to make with that, good for this pr but will open a separate pr for it as their is a significant refactoring that can be done.

Comment (maia-iyer) [reply frontend/src/components/tornjak-api-helpers.tsx:125]: I mean for the `axios.post` should it be an `axios.get`? the agents list api has become a get function in the v1 version

Comment (mamy-CS) [reply frontend/src/components/tornjak-api-helpers.tsx]: Oh, you are right about that, pushed. Thanks.

**Review by maia-iyer — APPROVED** [review]
LGTM! Thanks Mohammed!

**Pull request merged**